etc/profile-a-l/cargo.profile

```diff
@@ -0,0 +1,80 @@
+# Firejail profile for cargo
+# Description: The Rust package manager
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include cargo.local
+# Persistent global definitions
+include globals.local
+
+ignore noexec ${HOME}
+ignore noexec /tmp
+
+blacklist /tmp/.X11-unix
+blacklist ${RUNUSER}
+
+noblacklist ${HOME}/.cargo/credentials
+noblacklist ${HOME}/.cargo/credentials.toml
+
+# Allows files commonly used by IDEs
+include allow-common-devel.inc
+
+# Allow lua (blacklisted by disable-interpreters.inc)
```

Comment:
I can see the additional allow-foo.inc's are commented. Does cargo _ever_ need anything from lua, perl and/or python?

Reply:
lua,perl,python: if you project needs them
ssh: dependencies in private git repositories like

```
[dependencies.foo]
git = "ssh://git@github.com/bar/foo.git"
```

Reply:
Thanks for clearing that up. I'm happy with this now I understand.